tf2onnx/onnx_opset/tensor.py

```diff
@@ -1248,3 +1248,142 @@ def version_10(cls, ctx, node, **kwargs):
         target_dtype = TensorProto.INT64
         if seq_len_dtype != target_dtype:
             ctx.insert_new_node_on_input(node, "Cast", node.input[1], to=target_dtype)
+
+
+@tf_op("ReverseV2")
+class ReverseV2:
+    @classmethod
+    def version_9(cls, ctx, node, **kwargs):
+        raise NotImplementedError("ReverseV2 is not supported to convert in OPSET 9,"
+                                  " if possible please try using OPSET>=10 instead.")
+
+    @classmethod
+    def version_10(cls, ctx, node, **kwargs):
+
+        axes_node = node.inputs[1]
+        axes = axes_node.get_tensor_value() # Is a Python list.
+        len_axes = len(axes)
+
+        # Store input and output parameters of the ReverseV2 node.
+        rv2_in_names = [node.input[0]]
+        rv2_out_name = node.output
```

Comment:
why not use "node.output" in your code directly since the following code only reading rv2_out_name's value?